src/main.rs

```diff
@@ -44,8 +45,24 @@ pub fn main() {
 
     match builder_from_flags(bind_args.into_iter()) {
         Ok((builder, output)) => {
-            let mut bindings = builder.generate()
-                .expect("Unable to generate bindings");
+
+            let builder_result =
+                panic::catch_unwind(||
```

Comment:
could we move the "trying to write, etc" using `try!` inside the `catch_unwind`? That would avoid having to force `UnwindSafe`.